parquet-hadoop/src/main/java/org/apache/parquet/hadoop/util/HadoopPositionOutputStream.java

```diff
@@ -51,7 +51,9 @@ public void write(byte[] b, int off, int len) throws IOException {
   }
 
   public void sync() throws IOException {
-    wrapped.hsync();
+    if (wrapped.hasCapability("hsync")) {
+      wrapped.hsync();
```

Comment:
@steveloughran Does this make sense to you?

Reply:
hsync is actually really expensive. even on hdfs you shouldn't be calling it as it doesn't return until the data has been replicated and persisted.

S3A tells you off because some apps (hbase etc) really rely on sync to commit work. there's switch to actually fail...turning that on let us find which bits of code actually expected it to work.

Ideally code shouldn't be using outside of work whey want to be 100% confident data is written.

My IDE doesn't show any uses of the method...is there some through reflection?